ochazuke/webhooks/__init__.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""Flask Blueprint for our "webhooks" module.webhooks.
+
+See https://developer.github.com/webhooks/ for what is possible.
+"""
+
+import json
+import logging
+
+from flask import Blueprint
+from flask import request
+
+from ochazuke import app
+
+from .helpers import is_github_hook
+from .helpers import process_issue_event_info
+from .helpers import process_label_event_info
+from .helpers import process_milestone_event_info
+
+webhooks = Blueprint('webhooks', __name__, url_prefix='/webhooks')
+
+
+@webhooks.route('/issues', methods=['POST'])
+def issues_hooklistener():
+    """Listen for the "issues" webhook event.
+
+    By default, we return a 403 HTTP response.
+    """
+    if not is_github_hook(request):
+        return ('Move along, nothing to see here', 401, {'Content-Type':
```

Comment:
We use `{'Content-Type': 'text/plain'}` quite a bit in this file, perhaps we can make a constant and re-use it? 

`TEXT_PLAIN = {'Content-Type': 'text/plain'}`